ci-operator/step-registry/cucushift/installer/check/capability/cucushift-installer-check-capability-commands.sh

```diff
@@ -140,8 +140,7 @@ if [[ "${ADDITIONAL_ENABLED_CAPABILITIES}" != "" ]]; then
     enabled_capability_set="${enabled_capability_set} ${ADDITIONAL_ENABLED_CAPABILITIES}"
 else
     readarray -t additional_caps_from_config_array < <(yq-go r "${SHARED_DIR}/install-config.yaml" "capabilities.additionalEnabledCapabilities[*]")
-    additional_caps_from_config=${additional_caps_from_config_array[*]}
-    [[ -n "${additional_caps_from_config}" ]] && enabled_capability_set="${enabled_capability_set} ${additional_caps_from_config}"
+    [[ -n "${additional_caps_from_config_array}" ]] && enabled_capability_set="${enabled_capability_set} ${additional_caps_from_config_array[*]}"
```

Comment:
```suggestion
    [[ -n "${additional_caps_from_config_array[*]}" ]] && enabled_capability_set="${enabled_capability_set} ${additional_caps_from_config_array[*]}"
```

Reply:
@jinyunma the job failed again with the same error, actually I can not reproduce this issue on my local, I am even thinking it is a bash bug. 

Maybe you did not add `additional_caps_from_config_array=()` line again,  and change `*` to `@` for a luck.

Reply:
seems "@" is not passed on shell check, just add  `additional_caps_from_config_array=()" to see if the issue can be fixed.